main.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.console_entry_point
+end
+
+def list_of_options
+  puts
+  puts 'Choose an option by entering a number: '
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+end
+
+def option(input)
+  case input
+  when '1'
+    list_all_books
+  when '2'
+    list_all_people
+  when '3'
+    create_person
+  when '4'
+    create_book
+  when '5'
+    create_rental
+  when '6'
+    list_all_rentals
+  else
+    puts 'Please enter a number between 1 and 7.'
+  end
+end
+
```

Comment:
- [ ] Kindly add the **command for running the app** in your _ReadMe_, as requested by the previous code reviewer. This is can be of great benefit to non-technical users

Reply:
thanks for the feedback,

Reply:
but this is not a requirement

Reply:
@banlon-jones 
Kindly note that the above is a default requirement
cheers!